relayer/middleware/wallet/wallet-management.ts

```diff
@@ -107,6 +108,12 @@ function buildWalletsConfig(
           tokens: tokens[chainId] ?? [],
         });
       }
+    } else if (chainId === CHAIN_ID_SEI) {
+      for (const mnemonic of keys) {
```

Comment:
For this to work, we need this library to add support for Sei. This should happen soon, but until we are ready, we should just ignore sei for wallet-management (which at the moment is just wallet-monitoring really).
You can just `continue` in this case

Reply:
We want to add Sei to relayer engine so that we can run a single relayer with multiple Sei wallets loaded in. Do we not need Sei in wallet-management for this to work?

Reply:
wallet-management provides monitoring of wallet balances (so that we can monitor the balance of wallets to make sure they have enough funds to pay for a relay).
It's not strictly necessary, but it's something we should have.
Monitoring is been implemented in a [separate repo](https://github.com/xlabs/wallet-monitor) along with other wallet-management features we are working on. Until we add support for sei in that repo if you create a record for sei in this configuration map, monitoring will fail altogether (will throw an error saying that it can't handle Sei wallets)

Reply:
I'm ok with merging with a `continue` until we can support Sei on wallet-monitor. It shouldn't be long before that and it's very likely that we can have support for this before Sei launches

Reply:
Got it. I added a `continue` there and commented out the other section with a note to uncomment once Sei is ready for wallet-management.